src/components/Form/FormFields/Autocomplete.tsx

```diff
@@ -24,6 +24,7 @@ type AutocompleteFormFieldProps<T, V> = FormFieldBaseProps<V> & {
   optionValue?: OptionCallback<T, V>;
   optionDescription?: OptionCallback<T, string>;
   optionIcon?: OptionCallback<T, React.ReactNode>;
+  optionImage?: OptionCallback<T, string>;
```

Comment:
so you can make this allow to return `undefined`

```suggestion
  optionImage?: OptionCallback<T, string | undefined>;
```